src/main/java/com/avispl/dal/communicator/cisco/CiscoCommunicator.java

```diff
@@ -2537,7 +2629,7 @@ private void enrichVideoChannelStatsData(VideoChannelStats videoChannelStats, Ca
      *
      * @param statistics containing all the device statistics collected
      * @return true if device in presentation mode, false if not in presentation mode
-     * */
+     */
     private boolean checkPresentationMode(Map<String, String> statistics) {
         String presentationMode = statistics.get(CONFERENCE_PRESENTATION_MODE);
         if (StringUtils.isNotNullOrEmpty(presentationMode) && presentationMode.equalsIgnoreCase("On")) {
```

Comment:
Please change it with return StringUtils.isNotNullOrEmpty(presentationMode) && presentationMode.equalsIgnoreCase("On"); only